custom/ewsghana/management/commands/ews_verify_numbers.py

```diff
@@ -0,0 +1,96 @@
+from django.core.management import BaseCommand
+from corehq.apps.sms.mixin import apply_leniency, PhoneNumberInUseException, VerifiedNumber
+from corehq.apps.users.models import CommCareUser
+from custom.ilsgateway.models import ILSGatewayConfig
+from dimagi.utils.decorators.memoized import memoized
+
+
+TEST_BACKEND = 'MOBILE_BACKEND_TEST'
+
+
+class Command(BaseCommand):
+    """
+    Manually test the stock data migration.
+    """
+    args = '<domain>'
+
+    @memoized
+    def _get_logistics_domains(self):
+        from custom.ewsghana.models import EWSGhanaConfig
+        return ILSGatewayConfig.get_all_enabled_domains() + EWSGhanaConfig.get_all_enabled_domains()
+
+    def _save_verified_number(self, domain, user, connection):
+        backend_id = None
+        if connection['backend'] == 'message_tester':
+            backend_id = TEST_BACKEND
+        number = apply_leniency(connection['phone_number'])
+        try:
+            return user.save_verified_number(domain, number, True, backend_id=backend_id)
+        except PhoneNumberInUseException:
+            v = VerifiedNumber.by_phone(number, include_pending=True)
+            if v.domain in self._get_logistics_domains():
+                v.delete()
+                return user.save_verified_number(domain, number, True, backend_id=backend_id)
+
+    def check_backend(self, connection, vn):
+        if connection['backend'] == 'message_tester' and vn.backend_id != TEST_BACKEND:
+            vn.backend_id = TEST_BACKEND
+            vn.save()
+        if connection['backend'] == 'smsgh' and vn.backend_id is not None:
+            vn.backend_id = None
+            vn.save()
+
+    def handle(self, domain, *args, **options):
+
+        for sms_user in CommCareUser.by_domain(domain):
```

Comment:
how come we don't? WebUsers also have a property for `user_data` so I think it would be worth it to store the connection data the same way so we can check it at the end